tests/conftest.py

```diff
@@ -3,131 +3,401 @@
 """
 import os
 import pathlib
+import shutil
+import subprocess
 import tempfile
-from typing import Any, Dict, List, Tuple
+from typing import Any, Dict, Generator, List, Tuple
 
 import boto3
 import boto3.session
+import duckdb
 import pyarrow as pa
 import pytest
 from moto import mock_s3
 from moto.server import ThreadedMotoServer
 from pyarrow import csv, parquet
+from pycytominer.cyto_utils.cells import SingleCells
+
+from pycytominer_transform import column_sort
 
 
 # note: we use name here to avoid pylint flagging W0621
-@pytest.fixture(name="get_tempdir")
-def fixture_get_tempdir() -> str:
+@pytest.fixture(name="get_tempdir", scope="session")
+def fixture_get_tempdir() -> Generator:
     """
     Provide temporary directory for testing
     """
 
-    return tempfile.gettempdir()
+    tmpdir = tempfile.mkdtemp()
 
+    yield tmpdir
 
-@pytest.fixture()
-def data_dir_cellprofiler() -> str:
+    shutil.rmtree(path=tmpdir, ignore_errors=True)
+
+
+@pytest.fixture(name="data_dir_cellprofiler")
+def fixture_data_dir_cellprofiler() -> str:
     """
-    Provide a data directory for cellprofiler
+    Provide a data directory for cellprofiler test data
     """
 
     return f"{os.path.dirname(__file__)}/data/cellprofiler"
 
 
+@pytest.fixture(name="data_dirs_cytominerdatabase")
+def fixture_data_dirs_cytominerdatabase() -> List[str]:
+    """
+    Provide a data directory for cytominer-database test data
+    """
+
+    basedir = f"{os.path.dirname(__file__)}/data/cytominer-database"
```

Comment:
i recommend sticking to pathlib over os

Reply:
Great point, thanks. I'll update this line to use a pathlib-based approach instead. This also makes me wonder if adding a custom linting check to avoid `os.path` might help avoid this in the future. I'll investigate further here to see if this is something which could be added in a simple way with existing work.